supporting_content/attributes/discount_handling.md

```diff
@@ -0,0 +1,88 @@
+# Discount Types
+
+Providers offer various discounting schemes for their service offerings. These discounts typically fall into common types such as:
+
+* Commitment-based discounts
+* Tier-based discounts
+* Negotiated discounts
+* Promotional discounts
+* Partnership-based discounts
+
+## Commitment-based discounts
+
+Notes:
+
+* Usage-based commitment discounts
+* Spend-based commitment discounts
+
+## Tier-based discounts
```

Comment:
Is tier-based pricing (pricing user tiers) considered as type of discount? Indeed, one can see tier-based pricing as the implementation of a volume-based discount.

Otherwise, is "tier-based discount" referring to discounts applying to specific tiers? For example, a discount could be applied on the 1st Tier to implement a trial period.

Reply:
The thinking behind this tier-based discount is on the partnership level i.e. premier tier has more enterprise (or volume, etc) discounts than, say, advanced tier. But then again, we could also just consider this as under the partnership discounts. I wasn't really sure. Do you think it's better to remove? I'll add a note under Discussion / Scratch Space.

Reply:
wouldn't tier pricing be covered by usage based?